src/components/pagination/_pagination.scss

```diff
@@ -195,13 +195,16 @@ $css--helpers: true;
 }
 
 @mixin pagination--x {
+  $pagination-breakpoint: 530px;
```

Comment:
Is there a chance we could use a breakpoint defined in: https://github.com/IBM/carbon-components/blob/master/src/globals/scss/vendor/%40carbon/layout/scss/_breakpoint.scss#L16 or is this bespoke for pagination?

Also, should we express the media query in terms of `rem`?

Reply:
@joshblack i think this is bespoke for pagination, as the area for all of these elements being able to be squished together is kinda particular. it's optimized for basically displaying the standard fullscreen controls in as small a view as possible. i can test enlarging i tho! might give us a little more breathing room

Reply:
resolved this after our discussion with akira!